tests/core/test_errors.py

```diff
@@ -0,0 +1,26 @@
+# Standard Library
+import os
+import tempfile
+
+# First Party
+from smdebug.exceptions import InvalidCollectionConfiguration
+
+# Local
+from ..util import EnvManager
```

Comment:
nit: can you replace the relative import with `from tests.util import EnvManager`

Reply:
sure

Reply:
tests/core/test_errors.py:None (tests/core/test_errors.py)
ImportError while importing test module '/Users/pllarroy/devel/tornasole/sagemaker-debugger/tests/core/test_errors.py'.
Hint: make sure your test modules/packages have valid Python names.